### ISSUE TITLE
Running SUSHI Imports Returns Results for only January 1970

### ISSUE THREAD

**Issue by LibraryIT** [coral-erm/coral#602]
Whenever I try to run a Sushi import, regardless of the date I set, it only queries results for January 1970.  This leads to nothing being imported.  It seems to happen regardless of the vendor selected.  Otherwise the task seems to run fine.  It also produces a lot of errors in /var/log/httpd/error_log.  
 
See the following example (the IP address of the client is redacted, but in the log file it is the address of my workstation):

[Thu Sep 26 18:11:01.135220 2019] [php7:warn] [pid 25955] [client xxx.xxx.xxx.xxx:61418] PHP Warning:  fopen(/www/coral/usage/archive/1569535861.txt): failed to open stream: Permission denied in /www/coral/usage/admin/classes/domain/SushiService.php on line 807, referer: http://lib-web02.skidmore.edu/coral/usage/sushi.php

[Thu Sep 26 18:11:34.369593 2019] [php7:warn] [pid 10538] [client  xxx.xxx.xxx.xxx:61423] PHP Warning:  file_get_contents(archive/1569535871.txt): failed to open stream: No such file or directory in /www/coral/usage/admin/classes/common/Utility.php on line 123, referer: http://lib-web02.skidmore.edu/coral/usage/sushi.php

[Thu Sep 26 18:11:01.135267 2019] [php7:warn] [pid 25955] [client  xxx.xxx.xxx.xxx:61418] PHP Warning:  fwrite() expects parameter 1 to be resource, boolean given in /www/coral/usage/admin/classes/domain/SushiService.php on line 808, referer: http://lib-web02.skidmore.edu/coral/usage/sushi.php

[Thu Sep 26 18:11:34.369641 2019] [php7:notice] [pid 10538] [client  xxx.xxx.xxx.xxx:61423] PHP Notice:  Undefined offset: 1 in /www/coral/usage/uploadConfirmation.php on line 51, referer: http://lib-web02.skidmore.edu/coral/usage/sushi.php

[Thu Sep 26 18:11:34.376062 2019] [php7:notice] [pid 10538] [client  xxx.xxx.xxx.xxx:61423] PHP Notice:  Undefined variable: checkYear in /www/coral/usage/uploadConfirmation.php on line 347, referer: http://lib-web02.skidmore.edu/coral/usage/sushi.php

[Thu Sep 26 18:11:35.948559 2019] [php7:warn] [pid 10538] [client  xxx.xxx.xxx.xxx:61423] PHP Warning:  file_get_contents(archive/1569535871.txt): failed to open stream: No such file or directory in /www/coral/usage/admin/classes/common/Utility.php on line 123, referer: http://lib-web02.skidmore.edu/coral/usage/uploadConfirmation.php?importLogID=53

[Thu Sep 26 18:11:35.949851 2019] [php7:warn] [pid 10538] [client  xxx.xxx.xxx.xxx:61423] PHP Warning:  fwrite() expects parameter 1 to be resource, boolean given in /www/coral/usage/uploadComplete.php on line 961, referer: http://lib-web02.skidmore.edu/coral/usage/uploadConfirmation.php?importLogID=53

[Thu Sep 26 18:11:42.202562 2019] [php7:warn] [pid 30841] [client  xxx.xxx.xxx.xxx:61426] PHP Warning:  mktime() expects parameter 6 to be integer, string given in /www/coral/usage/uploadComplete.php on line 993, referer: http://lib-web02.skidmore.edu/coral/usage/uploadConfirmation.php?importLogID=52

**Comment by andyp-uk** [comment]
Have the permissions changed on the coral/usage/archive folder? What are they currently set to?

**Comment by LibraryIT** [comment]
I haven't changed the permissions lately.  They are 755 for that directory.  I also noticed the sushistore directory was owned by the apache user and not the regular user I log in as.    I changed it and don't get the error above, but instead see the following warning:

PHP Warning:  file_put_contents(/www/coral/usage/sushistore/EBSCO_DB1_201
8-07-01_2018-12-31.xml): failed to open stream: Permission denied in /www/coral/usage/admin/classes/domain/SushiService.php on line 533, referer: http://lib-web0
2.skidmore.edu/coral/usage/sushi.php

I changed the file to be part of the apache group, but that didn't help.  It also has permissions set to 755.
